pyproject.toml

```diff
@@ -1,3 +1,12 @@
 [build-system]
 requires = ["setuptools>=40.8.0", "wheel", "bleach>=2.1.0", "docutils>=0.13.1", "Pygments>=2.5.1"]
 build-backend = "setuptools.build_meta:__legacy__"
+
+[tool.mypy]
+warn_unused_configs = true
+[[tool.mypy.overrides]]
+# These modules do not yet have types available.
+module = [
+    "cmarkgfm.*"
+]
+ignore_missing_imports = true
```

Comment:
Instead of a blanket ignore, I prefer to explicitly ignore packages that are missing imports (e.g. in [twine/mypy.ini](https://github.com/pypa/twine/blob/c506b225b3ab57a40571001825dea0058a55807a/mypy.ini#L21-L42)).

Reply:
This is a difference in syntax between mypy.ini and pyprpject.toml - this flag only affects the modules specified in this overrides section.
I did a similar thing over in warehouse https://github.com/pypa/warehouse/blob/a35c7589d243a71753d9cb1f246989c4f566a95c/pyproject.toml#L18